SpatialGDK/Extras/fastbuild/uninstall.ps1

```diff
@@ -37,7 +37,7 @@ if ($service) {
 }
 
 # Manually remove marker file from the brokerage, since it's not possible to cleanly exit FBuildWorker.exe
-$brokerageFile = [System.IO.Path]::Combine($env:FASTBUILD_BROKERAGE_PATH, "main", "17", [System.Net.Dns]::GetHostName())
+$brokerageFile = [System.IO.Path]::Combine($env:FASTBUILD_BROKERAGE_PATH, "main", "19.windows", [System.Net.Dns]::GetHostName())
```

Comment:
Good catch - can you add a comment to the top of the file next to the version saying what you'll need to do to keep this line up to date?

Reply:
Will do!